### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by utleroy**
Tree displaying center

### CODE REVIEW

**Comment by utleroy** [comment]
@ncgonz My tree gets watered and grows straight up

**Comment by utleroy** [comment]
@ncgonz @zoeames I have everything functioning except the console log displaying the tree. Is there something that I am missing? I can't figure out how to console log the actual tree object??

**Comment by copypastedeveloper** [comment]
@utleroy, you should be able to write it to the console the same way you wrote it to inner html.  The text will be different, but the idea should be the same.

Also, digging the variable names.

**Comment by utleroy** [comment]
@ncgonz @zoeames Just pushed final commit for review.

File: main.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+function checkForProperFormat(rows, width) {
+  if(!rows || !width) {
+    alert("Both input fields need a value!");
+  } else {
+    buildTree(rows, width);
+  }
+}
+
+/*function for building the tree from the user input. Then calls the treeDisplay function*/
+function buildTree(rows, width) {
```
Comment (copypastedeveloper) [inline main.js]: I like the explicit nature and the breaking down things into components that you've done for the most part, but I think you may have taken it a little too far on this one.  :)  Having a function that only calls another function is probably just unnecessary code that will/could break, and another thing that has to be changed every time you change the parameters of the other function.  I'd suggest calling `treeDisplay` directly from line 22.

**Comment by copypastedeveloper** [comment]
@utleroy This looks pretty good.  Only the one comment around maybe a little bit too much breakdown of functions.  I've found that I take 3 passes to get something right.  The first is really ugly , the 2nd is way too far in the other direction, and the third is just right.  

> I'm like Goldilocks, but with code.

**Comment by utleroy** [comment]
@ncgonz I think that I have er fixed up.

**Comment by copypastedeveloper** [comment]
👍

**Pull request merged**